addons/l10n_es_edi_facturae/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import csv
+
+from odoo.tools import file_open
+from . import models
+
+
+def _edit_tax_types(env, template_data):
```

Comment:
I wonder if with the last changes in master, this is still needed: https://github.com/odoo/enterprise/pull/37359/files#diff-7ad7cbcb5f209cfd27e15c0ccfc529768d7867c5d446ab94381d5feaa2139ecaR97